### ISSUE TITLE
ERR_INTERNAL_ASSERTION error when building with precompiled Node v12.16.1 (works with precompiled Node v12.16.2)

### ISSUE THREAD

**Issue by aral** [nexe/nexe#749]
## To reproduce

On Node v12.16.1:

```sh
git clone https://source.small-tech.org/site.js/app.git site.js
cd site.js
npm i
npm run install-locally
site --version
```

## What should happen

Site.js should run and show its version number.

## What actually happens

The following error is shown:

```
internal/assert.js:14
    throw new ERR_INTERNAL_ASSERTION(message);
    ^

Error [ERR_INTERNAL_ASSERTION]: This is caused by either a bug in Node.js or incorrect usage of Node.js internals.
Please open an issue with this stack trace at https://github.com/nodejs/node/issues

    at assert (internal/assert.js:14:11)
    at prepareMainThreadExecution (internal/bootstrap/pre_execution.js:129:3)
    at internal/main/run_main_module.js:7:1 {
  code: 'ERR_INTERNAL_ASSERTION'
}
```

If you repeat the instructions under Node version 12.16.2, the build succeeds and Site.js runs as expected.

## Environment

- Platform(OS/Version): Pop!_OS 19.10 (~Ubuntu 19.10)
- Host Node Version: 12.16.1
- Target Node Version: 12.16.1
- Nexe version: 3.3.2
- Python Version: 2.7.17 & 3.7.5

**Comment by calebboyd** [comment]
👀

**Comment by aral** [comment]
@calebboyd This looks like it was due to a regression in an assert in `util.inspect` in Node v12.16.1 that was fixed in Node v12.16.2 (see https://github.com/nodejs/node/commit/f756c809e0).

This is not an issue with Nexe. Closing.
